build.sbt

```diff
@@ -773,29 +810,33 @@ lazy val crossVersionSharedSources: Seq[Setting[_]] =
     }
   }
 
-def commonScalacOptions(scalaVersion: String) =
+def commonScalacOptions(scalaVersion: String, isDotty: Boolean) =
```

Comment:
As an fyi, the `isDotty` setting is computed by string matching on `scalaVersion`. Though, it's not exposed as such, so there may be no way we can take advantage of it.

Reply:
Right, I just thought this was clearer than trying to figure out and replicate whatever logic sbt-dotty uses for `isDotty`, but I have no objection to using `.startsWith("0")` or something similar instead.